extensions/wakatime/src/today-summary.tsx

```diff
@@ -0,0 +1,23 @@
+import { useSummary } from "./hooks";
+
+import { MenuBarExtra, getPreferenceValues } from "@raycast/api";
+
+export default function Command() {
+  const summary = useSummary();
+  const preference = getPreferenceValues();
+  const asksSentence = preference.showAsksSentence;
+
+  function getTitle(): string {
+    return `${summary.data
+      ?.map(
+        ([key, range]) =>
+          key === "Today" &&
+          (range.cummulative_total.seconds === 0 && asksSentence
+            ? asksSentence
+            : `Today: ${range.cummulative_total.text}`)
```

Comment:
You should use the `getDuration` helper in the utils. For consistency as it's used in other places in the extension.

https://github.com/raycast/extensions/blob/3a019140df76f4ff0720117a9daebbd4b1021ea6/extensions/wakatime/src/utils/index.ts#L3

Reply:
OK